.github/workflows/main-ci.yml

```diff
@@ -0,0 +1,51 @@
+name: Lint & install charts
+
+on: pull_request
```

Comment:
Wouldn't it be better to also run this workflow "on main" and remove this part from the release process?
I think it's a bit redundant and that would save some release time.

Reply:
We discussed a lot with @ldevulder about that yesterday and we thought it was good to test again lint and installation before releasing. We agree tests are done for every PR so the release will go fine. We see it as a double check :smiley: 
@jimmykarily WDYT?

Reply:
I would also prefer that we run this on `main` and skip it on releases. I believe the release should only start when tests are passing (that is, on a green `main`). There is still some double checking, because we run the tests on PRs and then on `main` after merge. In most cases, merging a green branch shouldn't break `main` but there is still a chance this could happen if the branch wasn't rebased on `main` when tests run. So by moving this check on `main`, we catch that case early and not when we release.

Reply:
Ok let me change that.